.asset-template.json.hb

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "Block Coding",
+  "description": "Create games using a high-level, block-based visual programming language.\r\n\r\nIntended as an educational tool for learners in the earlier stages of their journey towards becoming game developers. This plugin lets you create your first games with high-level blocks, avoiding the immediate need to learn to code in GDScript. Building games in this way provides a gentle introduction to programming concepts and allows you to focus your efforts on becoming familiar with the rest of the Godot Editor UI.",
+  "category_id": "5",
+  "godot_version": "4.2",
+  "version_string": "{{ context.release.tag_name }}",
```

Comment:
The first two tags were called `v0.2.0` and `v0.1.0`. Personally I really dislike the `v` prefix and I think we should omit it from future tags.

Reply:
I cannot find a way with `handlebars` to strip a leading `v` but thanks to you pointing out the whole release object below, I replaced this with the release name.